### ISSUE TITLE
web: cannot start speak after flutterTts.stop()

### ISSUE THREAD

**Issue by BoHellgren** [dlutton/flutter_tts#208]
## 🐛 Bug Report

The following logic works on Android but not on web:
```
FlutterTts flutterTts = FlutterTts();
await flutterTts.speak(text1);
(after a few seconds)
await flutterTts.stop();
await flutterTts.speak(text2);
```
When running under Safari on an iPhone, the second speak() has no effect and text2 cannot be heard.
It works if you wait until all of text1 has been spoken, but not if you interrupt it.
(Doing a setVolume while text1 is being spoken also stops the second speak from working.)

### Expected behavior
Should work the same way on web as on Android.

### Reproduction steps
On an iPhone, start Safari and enter the address of the example program: 
https://dlutton.github.io/flutter_tts/#/
Fill the text line with text.
Press Play and, after a second, press Stop.
Press Play again. Silence.

### Configuration
iPhone XR
iOS 14.4

**Comment by ZMChoo** [comment]
Any solution for this issue at this moment? I'm still facing this issue now

**Comment by dlutton** [comment]
Still looking into this issue @ZMChoo .

**Comment by mono0926** [comment]
https://github.com/dlutton/flutter_tts/blob/48790bbe792050f8454343e30e547678bf0d3b15/example/lib/main.dart#L124 seems to be needed.

**Comment by dlutton** [comment]
fixed in 3.8.2
